package.json

```diff
@@ -24,6 +24,10 @@
   ],
   "author": "Postman Labs <help@getpostman.com> (=)",
   "license": "Apache-2.0",
+  "dependencies": {
+    "lodash": "4.17.4",
+    "postman-collection": "3.0.0"
```

Comment:
Instead of depending on collection, can we make be passed over? Then move this to Dev dependencies.

Reply:
Addressed.